Dockerfile

```diff
@@ -18,9 +18,9 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
+#RUN python manage.py migrate
```

Comment:
The line for running database migrations is commented out. If database migrations are required during the container startup, you should uncomment this line. Ensure that the database is accessible and properly configured before running migrations.